src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleController.java

```diff
@@ -35,10 +45,29 @@
     private void initialize() {
         listViewStyle();
         allMessages.itemsProperty().bind(errorViewModel.allMessagesDataproperty());
+        allMessages.scrollTo(errorViewModel.allMessagesDataproperty().getSize() - 1);
+        allMessages.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
+        errorViewModel.allMessagesDataproperty().addListener((ListChangeListener) (change -> {
+            int size = errorViewModel.allMessagesDataproperty().size();
+            if (size > 0) {
+                allMessages.scrollTo(size - 1);
+            }
+        }));
         descriptionLabel.setGraphic(IconTheme.JabRefIcon.CONSOLE.getGraphicNode());
     }
 
     @FXML
+    private void copySelectedLogEntries(KeyEvent event) {
+        KeyBindingPreferences keyPreferences = Globals.getKeyPrefs();
+        if (keyPreferences.checkKeyCombinationEquality(KeyBinding.COPY, event)) {
+            ObservableList<LogEvent> selectedEntries = allMessages.getSelectionModel().getSelectedItems();
+            if (!selectedEntries.isEmpty()) {
```

Comment:
Try to reuse the [copyLog](https://github.com/JabRef/jabref/blob/javafx/src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleViewModel.java#L67) method (i.e. create an overload which accepts a list of `LogEvent`s)